spec/models/feed_spec.rb

```diff
@@ -18,4 +18,17 @@
     feed.name = "somename"
     expect(feed.name).to eq("somename")
   end
+
+  it "is not expired when new" do
+    expect(feed.created_at).to be_nil
+    expect(feed.fetched_at).to be_nil
+    expect(feed.expired?).to be_falsey
+    expect { feed.save! }.to change { feed.created_at }
+    expect(feed.expired?).to be_falsey
```

Comment:
Using `be_false` and `be_true` in these new tests fails most checks; am I wrong for wanting `expired?` to strictly return a boolean?

Reply:
we can force it, but generally nil is as good as false and most people don’t bother